lib/ts/recipe/session/sessionAuth.tsx

```diff
@@ -35,7 +35,9 @@ type PropsWithAuth = {
 };
 
 type Props = (PropsWithoutAuth | PropsWithAuth) & {
+    requiredGrants?: Grant<any>[];
     onSessionExpired?: () => void;
+    onMissingGrant?: (grantKey: string) => void;
```

Comment:
Should we call it `grantId`? Cause we use that everywhere, for example, in the header response we use `missing-grant-id`